observability/logging/logging-6.0/log6x-release-notes.adoc

```diff
@@ -62,9 +62,25 @@ include::snippets/logging-support-exception-for-cluster-observability-operator-d
 
 * This enhancement updates the must-gather functionality to only capture information for inspecting Logging 6.0 components, including Vector deployments from ClusterLogForwarder.observability.openshift.io resources and the Red Hat managed LokiStack. (link:https://issues.redhat.com/browse/LOG-5949[LOG-5949])
 
-
 * This enhancement improves Azure storage secret validation by providing early warnings for specific error conditions. (link:https://issues.redhat.com/browse/LOG-4571[LOG-4571])
 
+* This enhancement updates the `ClusterLogForwarder` API to follow the Kubernetes standards. (link:https://issues.redhat.com/browse/LOG-5977[LOG-5977])
++
+.Example of a new configuration in the `ClusterLogForwarder` custom resource for the updated API
+[source,yaml]
```

Comment:
That is a 5.x example. We should use 6.x example
```
apiVersion: observability.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: <name>
spec:
  outputs:
  - name: <output_name>
    type: <output_type>
    <output_type>:
      tuning:
         deliveryMode: AtMostOnce
```

Reply:
Thank you. Applied.